MhLabs.AwsSignedHttpClient/Extensions/HttpClientExtensions.cs

```diff
@@ -25,6 +26,32 @@ public static class HttpClientExtensions
             return executionResult.Data;
         }
 
+        /// <summary>
+        /// Send request using httpClient
+        /// </summary>
+        /// <param name="client"></param>
+        /// <param name="method"></param>
+        /// <param name="path">with our without initial slash</param>
+        /// <param name="postData">default null</param>
+        /// <param name="contentType">default application/json</param>
+        /// <param name="cancellationToken">default default :)</param>
+        /// <returns><see cref="HttpResponseMessage"/></returns>
+        public static async Task<HttpResponseMessage> MhSendAsync(this HttpClient client, HttpMethod method, string path, object postData = null,
```

Comment:
MhSendAsync is not a good name since this is a public library, also unclear what the difference is from SendAsync just by looking at name

Reply:
Your're right ExecuteAsync will do the job just fine. We where a bit to focused on PostAsync. I'll close this PR 👍